src/common/rtc.cc

```diff
@@ -44,8 +44,8 @@ CudaModule::Chunk::Chunk(
       << "For lower version of CUDA, please prepend your kernel defintiions "
       << "with extern \"C\" instead.";
 #endif
-  std::vector<const char*> c_options(options.size());
-  for (const auto& i : options) c_options.emplace_back(i.c_str());
+  std::vector<const char*> c_options;
+  for (const auto& i : options) c_options.push_back(i.c_str());
```

Comment:
Why?

Reply:
This is a bug fix.  The original code path creates a vector twice the desired size.

Reply:
Oh, I see. Why not then just create the `c_options` with the right size and assign instead of `emplace_back` or `push_back`?

Reply:
As we discussed offline, this reinstates the prior working code.  It's simple and good enough for adding a small number of pointers to the vector.